loadable_apps/loadable_sample/wifiapp/binary_update.c

```diff
@@ -646,25 +650,52 @@ void binary_update_aging_test(void)
 {
 	int ret;
 	binary_update_info_t bin_info;
+#ifdef CONFIG_MMINFO
+	int fd;
+	struct mallinfo data;
+
+	fd = open(MMINFO_DRVPATH, O_RDWR);
+	if (fd <= 0) {
```

Comment:
In `open()` API description, it says
```
 *   The new file descriptor is returned on success; -1 (ERROR) is returned
 *   on any failure the errno value set appropriately.
```
So I think we don't need to check with `<= 0`.

Reply:
Right. but actually it returns a value which is larger than 2 on success.
That's because fd 0~2 is assigned for stdin/out/err when a thread is created.